setup.py

```diff
@@ -56,14 +56,16 @@
                    "Programming Language :: Python",
                    "Topic :: Scientific/Engineering"],
       url="https://github.com/pytroll/trollflow2",
-      packages=['trollflow2',
-                'trollflow2.plugins',
-                ],
+      packages=find_packages(),
+      # packages=['trollflow2',
+      #          'trollflow2.plugins',
+      #          ],
```

Comment:
why leave these comments ?

Reply:
I was not sure find_packages would find the plugins, that is why. Don't know what will need this?
Can we test/verify before I remove the old lines?